libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -124,10 +126,25 @@ singleton AP_Notify alias notify
 singleton AP_Notify method play_tune void string
 singleton AP_Notify method handle_rgb void uint8_t 0 UINT8_MAX uint8_t 0 UINT8_MAX uint8_t 0 UINT8_MAX uint8_t 0 UINT8_MAX
 
+include AP_Proximity/AP_Proximity.h
+
+singleton AP_Proximity alias proximity
+singleton AP_Proximity method num_sensors uint8_t
+singleton AP_Proximity method get_object_count uint8_t
+singleton AP_Proximity method get_closest_object boolean float'Null float'Null
+singleton AP_Proximity method get_object_angle_and_distance boolean uint8_t 0 UINT8_MAX float'Null float'Null
+
 include AP_RangeFinder/AP_RangeFinder.h
 
 singleton RangeFinder alias rangefinder
 singleton RangeFinder method num_sensors uint8_t
+singleton RangeFinder method distance_cm_orient uint16_t Rotation'enum Rotation::ROTATION_NONE Rotation::ROTATION_PITCH_7
```

Comment:
I think we probably only want to go as far as ROTATION_ROLL_90_PITCH_315 (one before ROTATION_PITCH_7)

Reply:
I just look at my PR, I did the same thing too, I'm not sure a ROTATION_PITCH_7 rangefinder would be valid for anything, it would probably work but nothing would use it.

Reply:
@IamPete1 I agree with you. I think that the maximum value is ROTATION_ROLL_90_PITCH_315.